src/utils/close-process.js

```diff
@@ -0,0 +1,87 @@
+import childProcess from 'child_process';
+
+const CHECK_PROCESS_IS_KILLED_TIMEOUT = 5000;
+const NEW_LINE_SEPERATOR_RE           = /(\r\n)|(\n\r)|\n|\r/g;
+
+const findProcessId = (browserId, psOutput) => {
+    const processIdRegex   = new RegExp('^\\s*(\\d+)\\s+.*' + browserId);
+    const lines            = psOutput.split(NEW_LINE_SEPERATOR_RE);
+
+    for (let i = 0; i < lines.length; i++) {
+        const match = processIdRegex.exec(lines[i]);
+
+        if (match)
+            return parseInt(match[1], 10);
+    }
+
+    return null;
+};
+
+const isProcessExist = (processId, psOutput) => {
+    const processIdRegex   = new RegExp('^\\s*' + processId + '\\s+.*');
+    const lines            = psOutput.split(NEW_LINE_SEPERATOR_RE);
+
+    return lines.some(line => processIdRegex.test(line));
+};
+
+const getProcessOutput = callback => {
+    const child = childProcess.spawn('ps', ['-eo', 'pid,command']);
+    let stdout  = '';
+    let stderr  = null;
```

Comment:
You can use `stderr = ''` here, and you won't need the `if` operator in the `data` event handler.